BoxContentSDKSampleApp/BoxContentSDKSampleApp/BOXSampleAppSessionManager.m

```diff
@@ -60,6 +61,23 @@ + (NSString *)rootCacheDirGivenSharedContainerId:(NSString *)sharedContainerId
     return containerURL.path;
 }
 
++ (NSString *)tempCacheDir
+{
+    NSString *cachesPath = [NSSearchPathForDirectoriesInDomains(NSCachesDirectory, NSUserDomainMask, YES) lastObject];
```

Comment:
`URLsForDirectory:inDomains:` might be recommended over this method - https://developer.apple.com/reference/foundation/1414224-nssearchpathfordirectoriesindoma?language=objc